### PR TITLE
Updated the readme to be more attractive

### PR DESCRIPTION
**Pull request by IndrekV**
### 📒 Description
I changed the format and made it more clear.
Moved the most important parts to the top.
Added some badges about our github progress (we can add more if we see that it would benefit us

### 🕶️ Types of changes
Readme update

### Things to do

- [x] Add screenshot to the top under the About section (all platforms side by side)
- [x] Add link to Microsoft store after release
- [x] Make the downloads section align to center

### CODE REVIEW

**Comment by skel35** [comment]
Microsoft Store ID is `9nk3rf9nbjnp`. 
Here is the badge generator https://developer.microsoft.com/en-us/store/badges.
HTML:
`<a href='//www.microsoft.com/store/apps/9nk3rf9nbjnp?cid=storebadge&ocid=badge'>English badge</a>`

**Comment by IndrekV** [comment]
@skel35 Everything should be ready now. Please look it over and merge if everything looks ok.

**Comment by skel35** [comment]
@IndrekV There's a piece of style code visible at the top
image

File: README.md [hunk 1]
```diff
@@ -1,27 +1,158 @@
...
+  <a href="#contribute">Contribute</a>
+</p>
+
+## About
+
+<table>
+<tr>
+<td>
+
+  **Toggl Desktop** is a Toggl time tracking client with many helper functionas that make tracking time more effortless and smooth. Features such as Idle detection, reminders to track and Pomodoro Timer make this app a great companion when productivity and efficiancy is the goal
```
Comment (skel35) [inline README.md]: typos

> function**a**s

> effici**a**ncy

File: README.md [hunk 2]
```diff
@@ -1,27 +1,158 @@
...
+  <tr align="center">
+    <td valign="top">
+      <p>Windows</p>
+      <br/>
+      <a class="link" href="https://toggl.github.io/toggldesktop/download/windows-stable/">32bit installer</a>
+      <br/>
+      <a class="link" href="https://toggl.github.io/toggldesktop/download/windows64-stable/">64bit installer</a>
+      <br/>
+      <a class="link" href="https://chocolatey.org/packages/toggl">Chocolatey</a>
+      <br/>
```
Comment (skel35) [inline README.md]: I would insert some space between Chocolatey link and Store badge as they look cramped. Maybe another linebreak.

Comment (skel35) [reply README.md]: Apparently, the style wasn't applied, and it should fix such issues.

**Comment by IndrekV** [comment]
Jep the styling is not supported the way the markdown showed me with the chrome markdown exstension. I'll work on a new approach. This is what I was aiming for.

Screenshot 2019-08-29 at 10 20 51

**Comment by IndrekV** [comment]
@skel35 This is ready for a final review. I made the design much more simple and clear.

**Comment by IndrekV** [comment]
I decided to drop all the badges all together so all download links are similar and clean.

File: README.md [hunk 3]
```diff
@@ -1,21 +1,68 @@
...
+<a href="https://toggl.github.io/toggldesktop/download/macos-stable/">64bit dmg</a>&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;<a href='https://itunes.apple.com/ee/app/toggl-desktop/id957734279?mt=12'>
+  Mac App Store</a>
+<br/>
+<br/>
+<i>Officially macOS 10.11 and newer stable macOS versions are supported.</i>
+
+## Windows
+
+<br/>
+<a href="https://toggl.github.io/toggldesktop/download/windows-stable/">32Bit installer</a>&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://toggl.github.io/toggldesktop/download/windows64-stable/">64Bit installer</a>&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://chocolatey.org/packages/toggl">Chocolatey</a>&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;<a href='//www.microsoft.com/store/apps/9nk3rf9nbjnp?cid=storebadge&ocid=badge'>Microsoft Store</a>
```
Comment (skel35) [inline README.md]: I'll put a 64bit link before a 32bit one and fix the capitalization of `bit` for consistency reasons.

**Pull request merged**